truewiki/views/edit.py

```diff
@@ -33,16 +33,20 @@ def save(user, old_page: str, new_page: str, content: str, payload) -> web.Respo
         body = source.create_body(wiki_page, user, "Edit", new_page=new_page, page_error=page_error)
         return web.Response(body=body, content_type="text/html")
 
+    # Truncate long edit messages
+    # The html textfield already limits this, but if someone wanted to they could modify the html
+    summary = (summary[:500] + "..(truncated)") if len(summary) > 500 else summary
+
     # If the old page doesn't exist, this creates a page. But it is possible
     # that this is done from an old page with a different name. Make sure the
     # new page always wins in this case.
     if not wiki_page.page_exists(old_page):
         create_new = True
         old_page = new_page
-        commit_message = f"new page: {old_page}"
+        commit_message = f"new page: {old_page} \n User Summary: {summary}" if summary else f"new page: {old_page}"
```

Comment:
We can deduplicate this a bit, by putting this outside the `if/else`  statement like this:

```
if summary:
  commit_message += f"\n User Summary: {summary}"
```

Reply:
Sure, easy fix.

Reply:
Owh, and really minor: it is common in git to do two `\n`s after the first line.